setup.py

```diff
@@ -39,7 +39,9 @@ def __str__(self):
         ['scarlet/operators_pybind11.cc'],
         include_dirs=[
             get_pybind_include(),
-            get_pybind_include(user=True)
+            get_pybind_include(user=True),
+            # Path to Eigen submodule
```

Comment:
Might be confusing to call this a submodule since it isn't actually a *git* submodule.